components/grid/export/events.md

```diff
@@ -283,7 +283,14 @@ The `OnAfterExport` event fires after the [OnBeforeExport](#onbeforeexport) even
 @code {
     private async Task OnExcelAfterExport(GridAfterExcelExportEventArgs args)
     {
-        excelStream = args.Stream;
+        var bytes = args.Stream.ToArray();
+        var excelStream = new MemoryStream(bytes);
+
+        //or
```

Comment:
We should explain what is the purpose/benefit of creating a new stream. I guess this may not be necessary in all scenarios.

Reply:
Updated